base/src/org/compiere/process/DunningPrint.java

```diff
@@ -155,46 +153,37 @@ else if (to.getEMail() == null || to.getEMail().length() == 0)
 				re = new ReportEngine(getCtx(), format, query, info);
 			boolean printed = false;
 			if (isEMailPDF()) {
-				EMail email = client.createEMail(to.getEMail(), null, null);
-				if (!email.isValid()) {
-					addLog (entry.get_ID(), null, null, 
-						"@RequestActionEMailError@ Invalid EMail: " + to);
-					internalError++;
-					continue;
-				}
 				if(text == null) {
 					continue;
 				}
 				text.setUser(to);	//	variable context
 				text.setBPartner(bp);
 				text.setPO(entry);
 				String message = text.getMailText(true);
-				if (text.isHtml()) {
-					email.setMessageHTML(text.getMailHeader(), message);
-				} else {
-					email.setSubject (text.getMailHeader());
-					email.setMessageText (message);
-				}
-				//
+				//	Get instance for notifier
+				DefaultNotifier notifier = (DefaultNotifier) QueueLoader.getInstance().getQueueManager(DefaultNotifier.QUEUETYPE_DefaultNotifier)
+						.withContext(getCtx())
+						.withTransactionName(get_TrxName());
+				//	Send notification to queue
+				notifier
+					.clearMessage()
+					.withApplicationType(DefaultNotifier.DefaultNotificationType_UserDefined)

```

Comment:
Hi! @yamelsenih , with this ApplicationType the withUserID() method should be mandatory.
As I see in this code block
https://github.com/adempiere/adempiere/blob/develop/base/src/org/spin/queue/notification/DefaultNotifier.java#L311

Reply:
Hello @yamelsenih , yes, with this change in the DefaultNotifier, this doesn't happen now